server/yarn.lock

```diff
@@ -105,6 +105,11 @@ ansi-align@^3.0.0:
   dependencies:
     string-width "^3.0.0"
 
+ansi-regex@^2.1.1:
```

Comment:
Consider updating this, to resolve the security error displayed on GitHub. If it keeps reinstalling itself, try to force it to be installed by including it in the `package.json`

Reply:
Doing yarn install keeps bringing back the older versions of ansi in the yarn.lock, I think one of our other dependencies is relying on an older ansi-regex